contracts/stellar-interchain-token/src/tests/test.rs

```diff
@@ -179,6 +183,10 @@ fn transfer() {
     assert_eq!(token.balance(&user1), amount);
 
     assert_auth!(user1, token.transfer(&user1, &user2, &600_i128));
+
+    let events = env.events().all();
+    goldie::assert!(format!("{:#?}", events));
```

Comment:
```suggestion
    goldie::assert!(format!("{:#?}", env.events().all()));
```
intermediate var doesn't seem necessary